packages/node/src/integrations/requestdata.ts

```diff
@@ -138,12 +139,13 @@ export class RequestData implements Integration {
   }
 }
 
-/** Convert `include` option to match what `addRequestDataToEvent` expects */
+/** Convert this integration's options to match what `addRequestDataToEvent` expects */
 /** TODO: Can possibly be deleted once https://github.com/getsentry/sentry-javascript/issues/5718 is fixed */
-function formatIncludeOption(
-  integrationInclude: RequestDataOptions['include'] = {},
-): AddRequestDataToEventOptions['include'] {
-  const { ip, user, ...requestOptions } = integrationInclude;
+function convertReqDataIntegrationOptsToAddReqDataOpts(
+  integrationOptions: Required<RequestDataIntegrationOptions>,
+): AddRequestDataToEventOptions {
+  const { transactionNamingScheme } = integrationOptions;
+  const { ip, user, ...requestOptions } = integrationOptions.include;
```

Comment:
l: Prefer this to be one call because it makes it easier for the reader to see that it's the same var (`integrationOptions`) under the hood.

```suggestion
  const {
    transactionNamingScheme,
    include: { ip, user, ...requestOptions },
  } = integrationOptions;
```

Reply:
Huh. Not sure I knew that you could do double extraction that way. TIL!